lib/fluent/plugin/buf_file.rb

```diff
@@ -56,6 +51,14 @@ def initialize
       def configure(conf)
         super
 
+        unless @path
```

Comment:
Buffer chunks have `unique_id`, which are generated with time and random value.
So it SHOULD NOT conflict with other chunks in other processes. No need to do such thing.